reframe/core/pipeline.py

```diff
@@ -1081,9 +1082,11 @@ def cleanup(self, remove_files=False, unload_env=True):
 
     def __str__(self):
         return ('%s (%s)\n'
-                '        tags: [%s], maintainers: [%s]' %
+                '        tags: [%s], maintainers: [%s]\n'
+                '        file: [%s]' %
```

Comment:
I prefer an output like the following:

```
  * tensorflow_wide_deep_check (found in /path/to/reframe/cscs-checks/apps/tensorflow/tensorflow_check.py)
        descr: Tensorflow official wide_deep test
        tags: [production], maintainers: [TM]
```

Reply:
Also for the tags, I think we can keep the default set formatting with the `{}`.